_Tests/AudibleApi.Tests/L0/ApiTests.Download.cs

```diff
@@ -297,19 +297,33 @@ public async Task unknown_status_code()
 		[TestMethod]
 		public async Task book_in_library()
 		{
+			var asin = "B07L162HDY";
+			var voucherJson = "{\"key\":\"00000000000000000000000000000000\",\"iv\":\"11111111111111111111111111111111\",\"refreshDate\":\"2021-06-21T19:52:32Z\",\"removalOnExpirationDate\":\"2021-08-25T19:52:32Z\",\"rules\":[{\"parameters\":[{\"expireDate\":\"2021-07-31T19:52:32Z\",\"type\":\"EXPIRES\"}],\"name\":\"DefaultExpiresRule\"},{\"parameters\":[{\"directedIds\":[\"amzn1.account.AAAAAAAAAAAAAAAAAAAAAAAAAAAA\"],\"type\":\"DIRECTED_IDS\"}],\"name\":\"AllowedUsersRule\"}]}";
+			var voucherObj = AudibleApiDTOs.VoucherDtoV10.FromJson(voucherJson);
+			voucherObj.Should().NotBeNull();
+
+			var license_response = AudibleApi.Tests.EncryptionHelper.EncryptVoucher(asin, voucherJson);
+			var decryptedVoucher = AudibleApi.Tests.EncryptionHelper.DecryptVoucher(asin, license_response);
+
+			decryptedVoucher.Should().BeEquivalentTo(voucherJson, "It was encrypted then decrypted with same parameters.");
+
+
+
 			var downloadLink = "ht" + "tps://dc.cloudfront.net/42/bk_2.aax?voucherId=cdn:6G&Policy=eQ__&Key-Pair-Id=AA";
 			var msg = $@"
-{{""content_license"":{{""acr"":""CRQ"",""asin"":""B07L162HDY"",""content_metadata"":{{""content_url"":{{""offline_url"":""{downloadLink}""}}}},""drm_type"":""Adrm"",""license_response"":""I="",""message"":""foo"",""request_id"":""66_SG"",""status_code"":""Granted"",""voucher_id"":""cdn:66_SG""}},""response_groups"":[""always-returned""]}}
+{{""content_license"":{{""acr"":""CRQ"",""asin"":""B07L162HDY"",""content_metadata"":{{""content_url"":{{""offline_url"":""{downloadLink}""}}}},""drm_type"":""Adrm"",""license_response"":""{license_response}"",""message"":""foo"",""request_id"":""66_SG"",""status_code"":""Granted"",""voucher_id"":""cdn:66_SG""}},""response_groups"":[""always-returned""]}}
```

Comment:
This would be another good candidate for "{asin}" since you abstracted it above.